connectors/v2/tunein.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '.container';
+
+Connector.artistTrackSelector = '.line1._navigateNowPlaying';
+
+Connector.isPlaying = function () {
+	return $('#tuner').hasClass('playing');
+};
```

Comment:
lol, I was told each connector should be a separate pull request for ease of testing.  Anyway you could add artwork here

```
Connector.trackArtImageSelector = '.image img';
```